sope-core/NGExtensions/FdExt.subproj/NSString+Encoding.m

```diff
@@ -97,14 +99,20 @@ + (NSStringEncoding)stringEncodingForEncodingNamed:(NSString *)_encoding {
 + (NSStringEncoding)stringEncodingForEncodingNamed:(NSString *)_encoding {
   NSString *s  = [_encoding lowercaseString];
   unsigned len = [s length];
-  
+
   if (s == nil)
     return 0;
 
   switch(len) {
+  case 0:
```

Comment:
Yes, I say for clarity. To begin the list with the very first value. If you dont agree with this aesthetic, just say and i will remove it.